cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
Right, these come along from time to time and we need to see where the discrepancy lies.
So, if CFME gives 100 Images.....
if you do something like.....
```python
len(provider.mgmt.list_images())
```
what do you get?